### ISSUE TITLE
[Bug] Infinite Refresh Loop on Login

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#7679]
When initially logging into Moped Test the page falls into an infinite refresh loop that does not stop. I can stop it by hitting "X" on the refresh button and clicking on a the Projects tab to reload the page.

Video shows:
- Looping behavior
- Errors in the console
- Stopping the loop 

https://images.zenhubusercontent.com/140626918/bfb589e7-3a25-47b5-a759-de1d9e137eb9/infinite_loop_login__1_.mp4

**Comment by patrickm02L** [comment]
Talked to @frankhereford who was having a similar but different issue with the infinite refresh in Test. 

>  got my problem fixed, and found a solution - adding myself to the seed data. 

After talking with Frank, we've determined that this is a band-aid of a hack to a much bigger problem around how Frank, Chia, Jorge, and I all share a database ID of 46. See PR [#456](https://github.com/cityofaustin/atd-moped/pull/456#pullrequestreview-800333138)

**Comment by amenity** [comment]
Another one, @sergiogcx - do you think this is something we need to fix for Moped 1.0, or tech debt we can look at down the line?

**Comment by amenity** [comment]
@frankhereford - are y'all still experiencing this? If so, we can re-appropriate this issue to address whatever database cleanup needs to be done.

**Comment by frankhereford** [comment]
No, I have not had a login problem, particularly an infinite loop that prevented me from logging in. I can't speak for everyone, but I have not heard anyone else voicing a problem related to this. I think closing this is the best route.
